Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,60 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+## Missing Annotations
+
+Dynamic isolation gives you tools to express isolation at runtime.
+But, you may also find you need to describe other concurrency properties
+that are missing from unmigrated modules.
+
+### Unmarked Sendable Closures
+
+A callback that is missing a `Sendable` annotation can be a problem when
+combined with actor-isolated types.
+
+```swift
+@MainActor
```

Comment:
Let's show the fictional signature:

```suggestion
extension JPKJetPack { 
  func jetPackConfiguration(_: callback: () ->()) { /*...*/ }
}

@MainActor
```

Reply:
You're right including this is important context.